SpatialGDK/Source/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -51,7 +51,7 @@ bool USpatialGameInstance::HasSpatialNetDriver() const
 	return bHasSpatialNetDriver;
 }
 
-bool USpatialGameInstance::StartGameInstance_SpatialGDKClient(FString& Error)
+bool USpatialGameInstance::StartGameInstance_SpatialGDKClient(bool bIsPIE, FString& Error)
```

Comment:
How about instead of `bool bIsPIE` we use the `#if WITH_EDITOR` macro?

Reply:
Because `WITH_EDITOR` isn't just for PIE. It is a preprocessor macro that determines when the code is compiled, and it will still run if you run the game via the editor but without actually opening the editor via `-game` or `-server`, e.g. in a local deployment with managed editor workers.